docs/SRS-Volere/SRS.tex

```diff
@@ -183,13 +183,87 @@ \subsection{Assumptions}
+also managed manually.
+
+\item Verification of work performed by contractors currently requires
+a physical visit to site. There is no automated process to collect information 
+such as photos of the work performed, other than manually through email.
+\item It is currently difficult to know whether the labour cost
+billed by the contractors on their invoices is reflective of the actual work
+performed, as that information is only recorded in a physical logbook at each
+site and is not electronic. A contractor could overcharge for the work and it
+would be difficult to prove that their bill is inaccurate.
+
+\end{itemize}
+
 \subsection{The Context of the Work}
-\lips
+The below figure illustrates a high-level context model of the adjacent systems:
+\begin{figure}[h]
+\centering
+\includegraphics[width=1\textwidth]{4G06A-context-model.png}
+\caption{Context model}
+\end{figure}
+
 \subsection{Work Partitioning}
-\lips
+\begin{tabular}{|l|l|}
+\hline
+\textbf{Event Name} & \textbf{Input and Output} \\
+\hline
+1. Contractor Login & Contractor credentials (in) \\
+\hline
+2. Contractor Authentication & GPS location (in) \\
+\hline
+3. Staff Login & Staff credentials (in) \\
+\hline
+4. Sign Document & unsigned document (in), signed document (out) \\
+\hline
+5. Upload Report & report file (in) \\
+\hline
+6. Export Reports & report specifications (in), report data (out)\\
+\hline
+7. View Station Entry Protocols & Station location (in), entry protocol (out)\\
+\hline
+8. Add Contractor to System & Contractor information (in) \\
+\hline
+9. View Contractor Summary & Contractor name (in), contractor data (out) \\
+\hline
+10. Certification Expiry & notice of document expiration to staff (out) \\
+\hline
+\end{tabular}
+
 \subsection{Specifying a Business Use Case (BUC)}
-\lips
+The below points identify the business use cases for each of the
+business events by number in section 6.3.\\
+
+\begin{enumerate}
+\item Contractor authenticates themselves to City systems, proving that they 
+are authorized to be provided City information.\\
+\item Contractor proves they are at a site for transparency when work is being 
+completed.\\
+\item Staff authenticates themselves as authorized to view private data.\\
+\item Sign a document for contractual or regulatory purposes.\\
+\item Documents are stored for record keeping and relevant stakeholders 
+are notified as necessary.\\
+\item Export data of a specific type specified by the user. For example, all 
+crane insections in the last year.\\
+\item View a procedure of how staff and contractors must access a specified 
+station.\\
+\item New contractors are frequently onboarded, and their information is 
+recorded by the facilities managers.\\
+\item Facilities managers commonly need to see which employees of a contractor 
+are due for their health and safety training, WSIB expiry, etc.\\
+\item When a piece of documentation has expired, the next version needs to be 
```

Comment:
Do new versions of documents always need to exist after the expiry? After expiry, I'd assume our system would remove the document entirely and then the city just reuploads. Is there not a case where no new document needs to be uploaded after expiry.

Reply:
This probably won't an automatic process. The onus to update the document should be on the city.  The best the system should do is probably flag that a document is expired. I do see some value in keeping previous versions of the document (for example in the case of an audit)

Reply:
Agreed, that use case is outlining that one of the actions the user would be doing is uploading a new document when the previous one expires.